### PR TITLE
add test and support for odd-sized textures

### PR DESCRIPTION
**Pull request by machinaut**

### CODE REVIEW

File: mujoco_py/tests/test_modder.py [hunk 1]
```diff
@@ -49,28 +49,28 @@ def test_textures():
     modder.set_rgb('g2', (0, 255, 0))
     modder.set_rgb('g3', (0, 0, 255))
     modder.set_rgb('g4', (255, 0, 255))
-    compare_imgs(sim.render(200, 200, camera_name="topcam"),
+    compare_imgs(sim.render(201, 205, camera_name="topcam"),
```
Comment (welinder) [inline mujoco_py/tests/test_modder.py]: hmm, the issue was with texture sizes, not render buffer sizes, right? why do we need to change these test images?

Comment (machinaut) [inline mujoco_py/tests/test_modder.py]: we don't! happy to change them back if you want.

in any case the test images need to be regenerated.

Comment (machinaut) [inline mujoco_py/tests/test_modder.py]: I would add that i think its _slightly_ better coverage to have width != height and to have odd numbers.

Comment (welinder) [inline mujoco_py/tests/test_modder.py]: yep, makes sense. let's go with these new images!

**Pull request merged**